docs/source/comparison.rst

```diff
@@ -33,7 +33,7 @@ We hope it helps you to choose an appropriate framework for the demand.
 |       +-----------------------------+-------------------+------------------------+-------------------+----------------------------------------------------+
 |       | Variable-length loops       | Y                 | Y (scan)               |                   |                                                    |
 |       +-----------------------------+-------------------+------------------------+-------------------+----------------------------------------------------+
-|       | Stateful RNNs [3]_          | Y                 |                        |                   |                                                    |
+|       | Stateful RNNs [3]_          | Y                 |                        | Y [7]_             |                                                    |
```

Comment:
Align the column splitter (by deleting one space after `[7]_`). Sphinx table is so inconvenient... :(